app/views/incomes/previousYears/UpdateIncomeDetailsView.scala.html

```diff
@@ -15,45 +15,60 @@
  *@
 
 @import uk.gov.hmrc.tai.viewModels.income.previousYears.UpdateHistoricIncomeDetailsViewModel
-@import includes.{cancelLink, ptaHeader}
+@import includes.cancelLink
 @import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 @import uk.gov.hmrc.tai.forms.income.previousYears.UpdateIncomeDetailsForm
-@import includes.forms.{submit, textArea}
+@import includes.forms.submit
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukCharacterCount
+@import uk.gov.hmrc.govukfrontend.views.html.components.Text
+@import uk.gov.hmrc.govukfrontend.views.html.components.Hint
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukErrorSummary
+@import uk.gov.hmrc.govukfrontend.views.html.components.ErrorSummary
+@import uk.gov.hmrc.govukfrontend.views.html.components.CharacterCount
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLabel
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcSectionCaption
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF, govukCharacterCount: GovukCharacterCount, govukErrorSummary: GovukErrorSummary)
 
 @(viewModel: UpdateHistoricIncomeDetailsViewModel, updateIncomeDetailsForm: Form[String])(implicit request: Request[_], user: controllers.auth.AuthedUser, messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
-
-@header = @{
- ptaHeader(
-  formForErrorSummary = Some(updateIncomeDetailsForm),
-  displayBackLink = true,
-  mainHeadingText = Messages("tai.income.previousYears.details.heading", viewModel.formattedTaxYear),
-  preHeadingText = Messages("tai.income.previousYears.details.preHeading")
- )
-}
-
 @mainTemplate(
     title = Messages("tai.income.previousYears.details.title", viewModel.formattedTaxYear),
+    pageTitle = Some(messages("tai.service.navTitle")),
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = None) {
+    backLinkId = "backLink",
+    backLinkContent = Some(Messages("tai.back-link.upper")),
```

Comment:
Should there be a backLinkUrl here? Looks like the old JS one is being used

Reply:
I wanted to revisit everything with once all views are done so we avoid infinite loops